feat: create 2022-01-25-Community-Call-importanza-design-system (#1134)
italia/developers.italia.it: _posts/it/2022-01-25-Community-Call-importanza-design-system-servizi-pubblici digitali.md

---
layout: post

> title: "Community Call: L’importanza di un design system per i servizi pubblici digitali"

Reviewer: ```suggestion
title: "Community call: L’importanza di un design system per i servizi pubblici digitali"
```

subtitle: Quali le sfide da affrontare? Ne parliamo il 9 febbraio!
date: 2022-01-25 12:00:00 +0200
tags: developers designers designsystem
categories: news
author: Michela Gabrieli
image: /assets/icons/logo-it.png
locale: it
---

> Vi siete mai chiesti perché alcuni siti sono incredibilmente facili da navigare, e ci fanno sentire a casa fin dal primo utilizzo. Per esempio i grandi e-commerce, i social network, il nostro sito preferito di streaming? Ciò avviene perché tutte le grandi aziende digitali definiscono librerie,  componenti in uno stile unitario e regole condivise - il *design system* -  in modo che questi semplici “mattoncini” possano essere ripetuti e assemblati, creando per gli utenti un’esperienza coerente e soddisfacente. 

Reviewer: La prima frase non mi torna, dovrebbe avere un punto interrogativo.

Potrebbe essere:
> Vi siete mai chiesti perché alcuni siti, per esempio i grandi ecommerce o il vostro sito preferito di streaming, sono incredibilmente facili da navigare, e ci fanno sentire a casa fin dal primo utilizzo?

Author: Per me va bene come proponi

> Vi siete mai chiesti perché alcuni siti sono incredibilmente facili da navigare, e ci fanno sentire a casa fin dal primo utilizzo. Per esempio i grandi e-commerce, i social network, il nostro sito preferito di streaming? Ciò avviene perché tutte le grandi aziende digitali definiscono librerie,  componenti in uno stile unitario e regole condivise - il *design system* -  in modo che questi semplici “mattoncini” possano essere ripetuti e assemblati, creando per gli utenti un’esperienza coerente e soddisfacente. 

Reviewer: ```suggestion
Vi siete mai chiesti perché alcuni siti, per esempio i grandi e-commerce o il vostro sito preferito di streaming, sono incredibilmente facili da navigare, e ci fanno sentire a casa fin dal primo utilizzo?
Ciò avviene perché tutte le grandi aziende digitali definiscono librerie,  componenti in uno stile unitario e regole condivise - il *design system* -  in modo che questi semplici “mattoncini” possano essere ripetuti e assemblati, creando per gli utenti un’esperienza coerente e soddisfacente. 
```

Non sarebbe bellissimo se ciò accadesse anche per tutti i siti e i servizi digitali della Pubblica Amministrazione?

> Proprio per rispondere a questo quesito, a partire dallo scorso dicembre [Designers Italia](https://designers.italia.it/) in collaborazione con Developers Italia ha avviato una revisione completa del design system del Paese, uno strumento utile a  creare siti e servizi coerenti, familiari e fruibili da tutti i cittadini. A breve sarà resa pubblica una roadmap di aggiornamento delle librerie dei componenti e dei [due modelli applicati](https://designers.italia.it/modelli/), [Comuni](https://designers.italia.it/modello/comuni/) e [scuole](https://designers.italia.it/modello/scuole/). Questa revisione affronta una serie di sfide peculiari, data la sua applicazione in ambito pubblico:

Reviewer: > in collaborazione con Developers Italia

Do we really want to keep Developers & Designers apart? Sounds too cold and formal to me

> Proprio per rispondere a questo quesito, a partire dallo scorso dicembre [Designers Italia](https://designers.italia.it/) in collaborazione con Developers Italia ha avviato una revisione completa del design system del Paese, uno strumento utile a  creare siti e servizi coerenti, familiari e fruibili da tutti i cittadini. A breve sarà resa pubblica una roadmap di aggiornamento delle librerie dei componenti e dei [due modelli applicati](https://designers.italia.it/modelli/), [Comuni](https://designers.italia.it/modello/comuni/) e [scuole](https://designers.italia.it/modello/scuole/). Questa revisione affronta una serie di sfide peculiari, data la sua applicazione in ambito pubblico:

Reviewer: ```suggestion
Proprio per rispondere a questo quesito, a partire dallo scorso dicembre [Designers Italia](https://designers.italia.it/) in collaborazione con Developers Italia ha avviato una revisione completa del design system del Paese, uno strumento utile a  creare siti e servizi coerenti, familiari e fruibili da tutti i cittadini. A breve sarà resa pubblica una roadmap di aggiornamento delle librerie dei componenti e dei [due modelli applicati](https://designers.italia.it/modelli/), [comuni](https://designers.italia.it/modello/comuni/) e [scuole](https://designers.italia.it/modello/scuole/). Questa revisione affronta una serie di sfide peculiari, data la sua applicazione in ambito pubblico:
```

- un design system pubblico non viene applicato da enti tutti uguali tra loro, ma da una serie di enti con le loro specificità, dimensione, ruolo e obiettivi. Diversamente dal privato, la coerenza delle interfacce **deve lasciare un ampio spazio alla personalizzazione, rimanendo nel perimetro dell’usabilità, dell’accessibilità e del rispetto delle norme**.

## Appuntamento al 9 febbraio

Il prossimo appuntamento per la community si terrà il **9 febbraio  alle ore 15.30** e sarà dedicato a queste sfide grazie al contributo di: 
- **Daniele Tabellini**, UI/UX designer, Dipartimento per la Trasformazione Digitale, che coordinerà nei prossimi mesi il lavoro di aggiornamento del design system del Paese,
- **Marco Liberati**, il main contributor della libreria Design React Kit, 

> - **Giacomo Grassi**, l’head of customer experience di INPS, che ha da poco affrontato la sua sfida nel lancio di SIrio, il design system dell’ente.

Reviewer: ```suggestion
- **Giacomo Grassi**, l’head of customer experience di INPS, che ha da poco affrontato la sua sfida nel lancio di Sirio, il design system dell’ente.
```
